.circleci/config.yml

```diff
@@ -202,83 +236,358 @@ jobs:
+          command: |
+            sudo .circleci/scripts/package-validation/debian \
+              /tmp/workspace/artifacts/influxdb3*arm64.deb
+  check_package_rpm:
+    machine:
+      image: ubuntu-2204:current
+      resource_class: arm.medium
+    parameters:
+      arch:
+        type: string
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - add_ssh_keys:
+          fingerprints:
+            - 3a:d1:7a:b7:57:d7:85:0b:76:79:85:51:38:f3:e4:67
+      - checkout
+      - run: |
+          AWS_ACCESS_KEY_ID=$TEST_AWS_ACCESS_KEY_ID \
+          AWS_SECRET_ACCESS_KEY=$TEST_AWS_SECRET_ACCESS_KEY \
+          .circleci/scripts/package-validation/redhat << parameters.arch >> /tmp/workspace/artifacts/influxdb3*.<< parameters.arch >>.rpm
+  sign-packages:
+    circleci_ip_ranges: true
+    docker:
+      - image: quay.io/influxdb/rsign:latest
+        auth:
+          username: $QUAY_RSIGN_USERNAME
+          password: $QUAY_RSIGN_PASSWORD
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - fc:7b:6e:a6:38:7c:63:5a:13:be:cb:bb:fa:33:b3:3c
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: |
+          # We need this base so that we can filter it out of our checksums
+          # output and if that changes at all we only need to update it here
+          WORK_DIR="/tmp/workspace/artifacts/"
+          for target in ${WORK_DIR}*
+          do
+            case "${target}"
+            in
+              # rsign is shipped on Alpine Linux which uses "busybox ash" instead
+              # of bash. ash is somewhat more posix compliant and is missing some
+              # extensions and niceties from bash.
+              *.deb|*.rpm|*.tar.gz|*.zip)
+                rsign "${target}"
+              ;;
+            esac
 
-  # Compile cargo "release" profile binaries for influxdb3 & data generator
-  #
-  # Uses the latest ci_image (influxdb/rust) to build a release binary and
-  # copies it to a minimal container image based upon `rust:slim-buster`. This
-  # minimal image is then pushed to `quay.io/influxdb/influxdb3:${BRANCH}`.
-  build_release:
+            if [ -f "${target}" ]
+            then
+              # Since all artifacts are present, sign them here. This saves Circle
+              # credits over spinning up another instance just to separate out the
+              # checksum job.
+              sha256sum "${target}" | sed "s#$WORK_DIR##" >> "/tmp/workspace/artifacts/influxdb3-edge.${CIRCLE_TAG}.digests"
+
+              # write individual checksums
+              md5sum    "${target}" | sed "s#$WORK_DIR##" >> "${target}.md5"
+              sha256sum "${target}" | sed "s#$WORK_DIR##" >> "${target}.sha256"
+            fi
+          done
+      - persist_to_workspace:
+          root: /tmp/workspace
+          paths:
+            - artifacts
+      - store_artifacts:
+          path: /tmp/workspace/artifacts
+  publish-packages:
+    docker:
+      - image: cimg/python:3.6
```

Comment:
`cimg/python:3.6` here and elsewhere appears to be EOL based on https://circleci.com/developer/images/image/cimg/python